sharktank/requirements.txt

```diff
@@ -1,16 +1,21 @@
+iree-turbine
+
 # Runtime deps.
-gguf==0.6.0
-numpy==1.26.3
+gguf>=0.6.0
+numpy>=1.26.3
+
+# Needed for gguf (TODO: check why transitive dep is missing)
+sentencepiece>=0.1.98,<=0.2.0
```

Comment:
I think this should be set: https://github.com/ggerganov/llama.cpp/blob/5c333e014059122245c318e7ed4ec27d1085573c/gguf-py/pyproject.toml#L21-L26

Need to debug a bit more. Logs from what I've tried: https://gist.github.com/ScottTodd/07029ddd75a409934bd52098231686f2

Note to self: try installing just gguf on its own and see what happens.

Reply:
Tested gguf, it's missing?

```bash
python3.11 -m venv 3.11.venv
source 3.11.venv/bin/activate
python -m pip install gguf
pip freeze
# gguf==0.10.0
# numpy==2.1.3
# PyYAML==6.0.2
# tqdm==4.67.0
```